.circleci/config.yml

```
@@ -0,0 +1,70 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine. See: https://circleci.com/docs/2.0/configuration-reference
+version: 2.1
+
+workflows:
+  build:
+    jobs:
+      - lint:
+          dir: ./promql
+      - test:
+          dir: ./promql
+      - lint:
+          dir: ./remote_write
+      - test:
+          dir: ./remote_write
```

This can be simplified with a matrix:

```suggestion
      - lint:
        matrix:
          parameters:
            dir:
              - promql
              - remote_write
            - test:
        matrix:
          parameters:
            dir:
              - promql
              - remote_write
```